.eslintrc.js

```diff
@@ -66,6 +67,108 @@ module.exports = {
+        "no-new-func": "error",
+        // https://eslint.org/docs/rules/no-new-wrappers
+        "no-new-wrappers": "error",
+        // https://eslint.org/docs/rules/no-octal-escape
+        "no-octal-escape": "error",
+        // https://eslint.org/docs/rules/no-proto
+        "no-proto": "error",
+        // https://eslint.org/docs/rules/no-return-await
+        "no-return-await": "error",
+        // https://eslint.org/docs/rules/no-script-url
+        "no-script-url": "error",
+        // https://eslint.org/docs/rules/no-self-compare
+        "no-self-compare": "error",
+        // https://eslint.org/docs/rules/no-sequences
+        "no-sequences": "error",
+        // https://eslint.org/docs/rules/no-unmodified-loop-condition
+        "no-unmodified-loop-condition": "error",
+        // https://eslint.org/docs/rules/no-useless-call
+        "no-useless-call": "error",
+        // https://eslint.org/docs/rules/wrap-iife
+        "wrap-iife": "error",
+        // https://eslint.org/docs/rules/no-nested-ternary
+        "no-nested-ternary": "error",
+        // https://eslint.org/docs/rules/switch-colon-spacing
+        "switch-colon-spacing": "error",
+        // https://eslint.org/docs/rules/no-useless-computed-key
+        "no-useless-computed-key": "error",
+        // https://eslint.org/docs/rules/rest-spread-spacing
+        "rest-spread-spacing": ["error"],
+        // https://eslint.org/docs/rules/no-implicit-coercion
+        "no-implicit-coercion": "error",
+        // https://eslint.org/docs/rules/no-multi-spaces
+        "no-multi-spaces": "error",
+        // https://eslint.org/docs/rules/semi
+        "semi": ["error", "always"],
+        // https://eslint.org/docs/rules/key-spacing
+        "key-spacing": ["error", {
+            "afterColon": true
+        }],
+        // https://eslint.org/docs/rules/keyword-spacing
+        "keyword-spacing": ["error", {
+            "before": true,
+            "after": true
+        }],
+        // https://eslint.org/docs/rules/comma-spacing
+        // Also requires one line code fix
+        "comma-spacing": ["error", {
+            "after": true
+        }],
+        // https://eslint.org/docs/rules/no-multiple-empty-lines
+        "no-multiple-empty-lines": "error",
+        //https://eslint.org/docs/rules/no-unneeded-ternary
+        "no-unneeded-ternary": "error",
+        //https://eslint.org/docs/rules/no-whitespace-before-property
+        "no-whitespace-before-property": "error",
+        // https://eslint.org/docs/rules/object-curly-newline
+        "object-curly-newline": ["error", {"consistent": true, "multiline": true}],
+        // https://eslint.org/docs/rules/object-property-newline
+        "object-property-newline": "error",
+        // https://eslint.org/docs/rules/brace-style
+        "brace-style": "error",
+        // https://eslint.org/docs/rules/no-multiple-empty-lines
+        "no-multiple-empty-lines": ["error", {"max": 1}],
```

Comment:
This rule is specified above without the `{"max": 1}` qualifier. Is that intended?

Reply:
Good catch, thanks! I've fixed it.